src/js/integrations/viewhierarchy.ts

```diff
@@ -0,0 +1,52 @@
+import type { Event, EventHint, EventProcessor, Integration } from '@sentry/types';
+import { logger } from '@sentry/utils';
+
+import { NATIVE } from '../wrapper';
+
+/** Adds ViewHierarchy to error events */
+export class ViewHierarchy implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'ViewHierarchy';
+
+  private static _fileName: string = 'view-hierarchy.json';
+  private static _contentType: string = 'application/json';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = ViewHierarchy.id;
+
+  /**
+   * @inheritDoc
+   */
+  public setupOnce(addGlobalEventProcessor: (e: EventProcessor) => void): void {
+    addGlobalEventProcessor(async (event: Event, hint: EventHint) => {
+      const hasException = event.exception && event.exception.values && event.exception.values.length > 0;
+      if (!hasException) {
+        return event;
+      }
+
+      let viewHierarchy: Uint8Array | null = null;
+      try {
+        viewHierarchy = await NATIVE.fetchViewHierarchy()
+      } catch (e) {
+        logger.error('Failed to get view hierarchy from native.', e);
+      }
+
+      if (viewHierarchy) {
+        hint.attachments = [
```

Comment:
H: `attachmentType` should be `event.view_hierarchy`

Reply:
Thanks.

Reply:
This comment was resolved but I didn't see any change related to `attachmentType`.

Reply:
I remember fixing it... strange. Now it's here.